examples/campaign_management/get_campaigns_by_label.rb

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example gets campaigns by label. To add campaigns, run add_campaigns.rb.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def get_campaigns_by_label(customer_id, label_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  query = <<~EOD
+    SELECT campaign.id, campaign.name, label.id, label.name
+    FROM campaign_label WHERE label.id = '#{label_id}' ORDER BY campaign.id
+  EOD
+
+  ga_service = client.service(:GoogleAds)
+  response = ga_service.search(
+      customer_id,
+      query,
+      page_size: PAGE_SIZE)
+
+  response.each do |row|
+    puts sprintf("Campaign with ID %d and name '%s' was found.",
```

Comment:
We've been using #{} rather than sprintf in newer examples. Eventually we'll go back through and update all of them to be consistent, but no reason to add more future work now.